src/query/terms.js

```diff
@@ -53,6 +53,10 @@ export class Records extends TermBase {
     return new this.constructor(oqe('filter', this.expression, andExpression));
   }
 
+  sort(sortExpressions) {
```

Comment:
What do you think about the function name? I think I've seen at least `sortBy` and `orderBy` in the gitter/github discussions related to OQL.